express-middlewares/src/index.js

```diff
@@ -0,0 +1,22 @@
+const express = require('express');
+
+const port = 3000;
+
+const app = express();
+
+app.use((req, res, next) => {
+  res.set('middleware-header', req.url);
```

Comment:
Should you not parse `req.url`? It may have extra `/` slashes or something else which might not be required in response header.